api/src/contracttest/java/org/pmiops/workbench/consumer/AppsApiTest.java

```diff
@@ -46,16 +48,21 @@ class AppsApiTest {
       newJsonBody(
           body -> {
             body.stringType("appType", AppType.RSTUDIO.name());
-            body.stringType("allowedChartName", LeonardoAllowedChartName.RSTUDIO.name());
+            body.stringType("allowedChartName", LeonardoAllowedChartName.RSTUDIO.toString());
             body.object("labels", labels -> labels.stringType("key1", "value1"));
             body.stringType("descriptorPath");
             body.object(
                 "diskConfig",
                 diskConfig -> {
-                  diskConfig.stringType("diskType", LeonardoDiskType.SSD.name());
+                  diskConfig.stringType("diskType", LeonardoDiskType.SSD.toString());
                   diskConfig.numberType("size");
+                  diskConfig.stringType("name", "mock-disk");
+                });
+            body.object(
+                "customEnvironmentVariables",
+                customEnvironmentVariables -> {
+                  customEnvironmentVariables.stringType("key1");
                 });
```

Comment:
nit: Don't need the curly braces here: 
```suggestion
                customEnvironmentVariables -> customEnvironmentVariables.stringType("key1"));
```